pkg/agent/proxy/proxier_test.go

```diff
@@ -2184,9 +2175,11 @@ func testServiceInternalTrafficPolicyUpdate(t *testing.T,
 	fp.serviceChanges.OnServiceUpdate(svc, updatedSvc)
 	groupIDLocal := fp.groupCounter.AllocateIfNotExist(svcPortName, true)
 
-	mockOFClient.EXPECT().InstallEndpointFlows(bindingProtocol, gomock.InAnyOrder(expectedLocalEps)).Times(1)
 	mockOFClient.EXPECT().UninstallEndpointFlows(bindingProtocol, expectedRemoteEps).Times(1)
+	mockOFClient.EXPECT().UninstallServiceGroup(groupID).Times(1)
```

Comment:
The line validates the 4th bug "After updating InternalTrafficPolicy for a ClusterIP Service, the stale group is not removed" is fixed.